fw-child/template/menu-overlay.php

```diff
@@ -124,13 +124,13 @@
 					<div>
 						<a href="https://www.linkedin.com/company/climate-data-canada/" target="_blank" class="text-secondary me-2" title="LinkedIn"><i class="fab fa-linkedin"></i></a>
 						
-						<a href="https://twitter.com/ClimateData_ca" target="_blank" class="text-secondary me-2" title="X (Twitter)">𝕏</a>
+						<a href="https://twitter.com/ClimateData_ca" target="_blank" class="text-secondary me-2" title="X (Twitter)"><i class="fa-brands fa-x-twitter"></i></a>
 						
 						<a href="https://www.facebook.com/ClimateData.ca.Donneesclimatiques.ca" target="_blank" class="text-secondary me-2" title="Facebook"><i class="fab fa-facebook"></i></a>
 						
 						<a href="https://www.instagram.com/climatedata.ca/" target="_blank" class="text-secondary me-2" title="Instagram"><i class="fab fa-instagram"></i></a>
 						
-						<a href="https://soundcloud.com/climate-data-canada" target="_blank" class="text-secondary me-2" title="Instagram"><i class="fab fa-soundcloud"></i></a>
+						<a href="https://soundcloud.com/climate-data-canada" target="_blank" class="text-secondary me-2" title="Soundcloud"><i class="fab fa-soundcloud"></i></a>
```

Comment:
Should be "SoundCloud"

Reply:
@domcode-ew this is fixed and I also removed the extra margin that was causing the soundcloud icon to overflow to another line.